examples/ct/handlers.go

```diff
@@ -159,6 +159,8 @@ type LogContext struct {
 	trustedRoots *PEMCertPool
 	// rejectExpired indicates whether certificate validity period should be used during chain verification
 	rejectExpired bool
+	// extKeyUsages contains the list of EKUs to use during chain verification
+	extKeyUsages []x509.ExtKeyUsage
```

Comment:
Sorry to be a pain, but I think we've reached the point where this stuff would be better in a separate sub-structure.  If we added something like:
```go
type CertValidationOpts struct {
 	// trustedRoots is a pool of certificates that defines the roots the CT log will accept
￼  	trustedRoots *PEMCertPool
￼  	// rejectExpired indicates whether certificate validity period should be used during chain verification
￼  	rejectExpired bool
￼ 	// extKeyUsages contains the list of EKUs to use during chain verification
￼ 	extKeyUsages []x509.ExtKeyUsage
}
```
we could use that everywhere (as a field in `LogContext` and as parameter to `ValidateChain`)